src/modules/api/user/link/link.controller.ts

```diff
@@ -1,7 +1,12 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Post, Body, Param } from '@nestjs/common';
 import { LinkService } from './link.service';
 
 @Controller('users/:userId/links')
 export class LinkController {
-  constructor(private linkService: LinkService) {}
+  constructor(private linkService: LinkService) { };
+  @Post()
+  async create(@Body() body, @Param() params) {
```

Comment:
You are using single param. So, request it by passing the param name
@Param('userId')